ulauncher/config.py

```diff
@@ -15,8 +15,10 @@
 HOME = os.path.expanduser("~")
 CONFIG = os.path.join(os.environ.get("XDG_CONFIG_HOME", f"{HOME}/.config"), "ulauncher")
 USER_DATA = os.path.join(os.environ.get("XDG_DATA_HOME", f"{HOME}/.local/share"), "ulauncher")
+DATA_DIRS = [os.path.join(p, "ulauncher") for p in os.environ.get("XDG_DATA_DIRS", "").split(os.path.pathsep)]
 STATE = os.path.join(os.environ.get("XDG_STATE_HOME", f"{HOME}/.local/state"), "ulauncher")
 EXTENSIONS = os.path.join(USER_DATA, "extensions")
```

Comment:
I know that it means more code changes, but if we want to make this change, this variable should be more descriptive. As it is, the reader needs read the other variable, `EXTENSIONS_ALL`, to get a hint about what this variable isn't, but even then it isn't very descriptive other than knowing that it "doesn't contain all the extensions". Instead, this should be `MUTABLE_EXTENSIONS` or something like that.